### PR TITLE
lib: pdn: improve pdn_dynamic_params_get()

### PR DESCRIPTION
**Pull request by tomi-font**
Make use of nrf_modem_at_scanf() and add a changelog entry.

### CODE REVIEW

**Comment by lemrey** [comment]
Because this PR updates the changelog file, it must be reviewed by a tech writer before it can be merged.
I am adding back the `doc-required` label.

The `doc-required` label is added automatically when changes to certain files are detected (documentation changes), so in general it should not be removed. Instead you must add a tech writer to review and approve your PR.

The `changelog-entry-required` label is added automatically to all PRs. Some PRs don't require updates to the changelog, so that label can be removed, but if you remove it then you should provide an argument as to why the changelog does not need to be updated.

**Review by MarkusLassila — APPROVED** [review]

**Comment by tomi-font** [comment]
> Because this PR updates the changelog file, it must be reviewed by a tech writer before it can be merged.
I am adding back the doc-required label.
>
> The doc-required label is added automatically when changes to certain files are detected (documentation changes), so in general it should not be removed. Instead you must add a tech writer to review and approve your PR.

True. I know how it works but somehow dismissed it as a `changelog-entry-required` label, thinking that there were no documentation changes. But the changelog itself is a document.
Thanks again for pointing this out.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -529,6 +529,7 @@ Modem libraries
 
 * :ref:`pdn_readme` library:
 
+   * Added a function to retrieve dynamic parameters of an active PDN connection, :c:func:`pdn_dynamic_params_get`.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:532]:
```suggestion
   * Added the :c:func:`pdn_dynamic_params_get` function to retrieve dynamic parameters of an active PDN connection.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by lemrey — APPROVED** [review]
No problem thanks for the patch

**Review by SeppoTakalo — APPROVED** [review]

**Pull request merged**